content/agent/agent_checks.md

```diff
@@ -1,13 +1,15 @@
 ---
 title: Writing an Agent Check
 kind: documentation
+customnav: agentnav
 aliases:
   - /guides/agent_checks/
 ---
 
 ## Overview
 
-This page details how to collect metrics and events from a new data source by writing an Agent Check, a Python plug-in to the Datadog Agent. This page looks first at the `AgentCheck` interface, and then propose a simple Agent Check that collects timing metrics and status events from HTTP services.
+This page details how to collect metrics and events from a new data source by writing an Agent Check.  
+This page looks first at the `AgentCheck` interface, and then propose a simple Agent Check that collects timing metrics and status events from HTTP services.
```

Comment:
This page first looks at the `AgentCheck` interface, and then proposes a simple Agent Check that collects timing metrics and status events from HTTP services.

Reply:
👍